presto-orc/src/main/java/io/prestosql/orc/OrcWriter.java

```diff
@@ -106,7 +107,7 @@
     private final int stripeMaxRowCount;
     private final int rowGroupMaxRowCount;
     private final int maxCompressionBufferSize;
-    private final Map<String, String> userMetadata;
```

Comment:
Let's keep this final and change it to `new HashMap<>()`, then we can just update it normally in the update method